incubator/aci/create-k8-connector/script.md

```diff
@@ -31,11 +31,17 @@ CLI using the instructions below.  Find your subscriptionId with the
 az CLI:
 
 Use az to create a Service Principal that can perform operations on
-your resource group:
+your subscription:
 
 ```
 SUBSCRIPTION_ID=$(az account show | jq -r '.id')
-SP_JSON=$(az ad sp create-for-rbac --role="Contributor" --scopes="/subscriptions/$SUBSCRIPTION_ID/resourceGroups/$SIMDEM_RESOURCE_GROUP")
+SP_JSON=$(az ad sp create-for-rbac --role="Contributor" --scopes="/subscriptions/$SUBSCRIPTION_ID")
```

Comment:
Is this really necessary? This is a very powerful SP.

Reply:
We are blocked on the dev side so gonna need this to go through for now :/ - it will stay in incubator but this is what sean is going to end up demoing @seanmck is this okay with you?

Reply:
OK in incubator with a FIXME, not sure we want to be recommending it to folks though.